frontend/src/components/questions/UploadQuestion.vue

```diff
@@ -1,5 +1,226 @@
+<script setup lang="ts">
+import { computed, ref } from 'vue';
+import type { ComputedRef, Ref } from 'vue';
+import type { Quizoot } from '@interfaces/quizoot';
+
+interface Specs {
```

Comment:
We are the one who usually use long names.. Except when you really need it 😂 😂 

`UploadQuestionProps` might be better here.

Reply:
Argh, poor me 😭